src/Api.js

```diff
@@ -89,13 +95,97 @@ class Api {
 	}
 
 	/**
-	 * Get the WhoColor data for a given wiki page.
+	 * Extract token and user IDs from a WikiWho span element with `id='token-X'` and
+	 * `class='token-editor-Y'` attributes.
 	 *
-	 * @param  {string} wikiUrl URL of the wiki page that we want to analyze.
+	 * @param {HTMLElement} element
+	 * @return {int[]|boolean[]} A two-element array: 0 = token_id, 1 = user_id.
+	 */
+	getIdsFromElement( element ) {
+		const out = [ false, false ],
+			tokenMatches = element.getAttribute( 'id' ).match( /token-(\d+)/ ),
+			editorMatches = element.getAttribute( 'class' ).match( /token-editor-(\d+)/ );
```

Comment:
you could use `element.id` and `element.className` instead of using `getAttribute`

Reply:
Done.